perun-openapi/openapi.yml

```diff
@@ -10876,6 +10892,7 @@ paths:
           $ref: '#/components/responses/ExceptionResponse'
       parameters:
         - $ref: '#/components/parameters/ownerId'
+        - { name: force, description: "If true, use force deletion.", schema: { type: boolean },  in: query, required: false }
```

Comment:
The `deleteOwner` method doesn't support the `force` flag. You should probably also implement it in the `OwnersManagerMethod`.